mlir/utils/jenkins/Jenkinsfile

```diff
@@ -812,7 +813,7 @@ pipeline {
                 axes {
                     axis {
                         name 'CHIP'
-                        values 'gfx908', 'gfx90a', 'gfx1030', 'gfx1101'
+                        values 'gfx908', 'gfx90a', 'gfx942', 'gfx1030', 'gfx1101'
```

Comment:
Can you kick nightly run on this pull request with only MI300 enabled and see if performance reports are being generated correctly or not ?